core/src/test/java/org/elasticsearch/index/query/FuzzyQueryBuilderTests.java

```diff
@@ -107,7 +113,12 @@ public void testUnsupportedFuzzinessForStringType() throws IOException {
     }
 
     public void testToQueryWithStringField() throws IOException {
-        assumeTrue("test runs only when at least a type is registered", getCurrentTypes().length > 0);
+        try {
+            assumeTrue("test runs only when at least a type is registered", getCurrentTypes().length > 0);
```

Comment:
shall we move to using `@ConditionalIgnore` here. If I understood correctly in that case before and after wouldn't be executed for ignored methods, hence you wouldn't have to keep track of whether the test ran yourself